lib/wahwah.rb

```diff
@@ -45,6 +45,8 @@
 require 'wahwah/asf_tag'
 require 'wahwah/flac_tag'
 
+require 'stringio'
```

Comment:
`require 'stringio'` need to add on top of the file. Cause there are others file depends on StringIO.

Reply:
order doesn't really matter here because the symbol doesn't need to be defined at load time, only at runtime (I tested it). but changed it anyway!